README.md

```diff
@@ -290,6 +294,48 @@ security resources as depicted below. However, we plan to support all [custom se
 
 ![](docs/images/operator/starboard-operator.png)
 
+### Deployment
+
+#### With Static YAML Manifests
+
+You can install the operator with provided static YAML manifests with fixed values. However, this approach has its
+shortcomings. For example, if you want to change the container image or modify configuration parameters, you have to
+create new manifests or edit existing ones.
+
+To deploy the operator in the `starboard-operator` namespace and configure it to watch the `default`
+namespace:
+
+1. Send the definition of the [vulnerabilityreports][vulnerabilityreports-crd] custom resource to the Kubernetes API:
+
+   ```
+   $ kubectl apply -f deploy/crd/vulnerabilityreports.crd.yaml
+   ```
+2. Send the following Kubernetes objects definitions to the Kubernetes API:
+
+   ```
+   $ kubectl apply -f deploy/static/01-starboard-operator.ns.yaml \
+       -f deploy/static/02-starboard-operator.sa.yaml \
+       -f deploy/static/03-starboard-operator.clusterrole.yaml \
+       -f deploy/static/04-starboard-operator.clusterrolebinding.yaml
+   ```
+3. Create the `starboard-operator` deployment in the `starboard-operator` namespace to run the operator's container:
+
+   ```
+   $ kubectl apply -f deploy/static/05-starboard-operator.deployment.yaml
+   ```
+
+#### With Helm
+
+Helm, which is de facto standard package manager for Kubernetes, allows installing applications from parameterized
+YAML manifests called Helm charts. To address shortcomings of static YAML manifest we'll provide a Helm chart to deploy
+the Starboard operator. See [#187](https://github.com/aquasecurity/starboard/issues/187) to check the progress on that.
```

Comment:
I suggest to let the static resources be generated from the Helm chart in the future. See #211

Reply:
I totally agree @consideRatio Once we merge the Helm chart, we'll remove static manifests from `./deploy/static/*.yaml` and update the docs to use the `helm template` command for those who cannot use Helm CLI